### PR TITLE
NEOS 15

### PR DESCRIPTION
**Pull request by adeebshihadeh**

### CODE REVIEW

File: devices/eon/ramdisk_common/comma.sh [hunk 1]
```diff
@@ -15,23 +15,26 @@ fi
...
 echo 0 > /sys/class/leds/button-backlight/max_brightness
 
-# constrain everything but us to one cpu
-echo 0 > /dev/cpuset/background/cpus
-echo 0 > /dev/cpuset/system-background/cpus
-echo 0 > /dev/cpuset/foreground/boost/cpus
-echo 0 > /dev/cpuset/foreground/cpus
+# clock all four cores at historically supported frequencies
+# can be reset/overridden later in userspace if needed
+echo "performance" > /sys/devices/system/cpu/cpu0/cpufreq/scaling_governor
```
Comment (geohot) [inline devices/eon/ramdisk_common/comma.sh]: Why is this still in here?

Comment (adeebshihadeh) [reply devices/eon/ramdisk_common/comma.sh]: It shouldn't be, are you looking at an old commit?

File: devices/eon/home/.bashrc [hunk 2]
```diff
@@ -10,6 +10,3 @@ alias more='less'
 
 # everyone wants to start here anyway
 [ -d "/data/openpilot" ] && cd /data/openpilot
-
-# support for optional community-supported shell extensions
-[ -f "/data/community/.bashrc" ] && source /data/community/.bashrc
```
Comment (AskAlice) [inline devices/eon/home/.bashrc]: Why was this removed??? We will continue to modify the system partition on [emu-sh/.oh-my-comma](https://github.com/emu-sh/.oh-my-comma)

File: devices/eon/build_usr/install.py [hunk 3]
```diff
@@ -183,7 +183,6 @@ def install_package(pkg_deps, pkg_filenames, pkg):
         'wget',
         'xz-utils',
         'zlib-dev',
-        'zsh',
```
Comment (AskAlice) [inline devices/eon/build_usr/install.py]: Why was this removed? It will make it a lot easier for my code to rationalize modifying the system partition to install powerline via a python module

**Review by AskAlice — COMMENTED** [review]
It seems the two changes I talked to jyoung about to prevent [emu.sh](https://github.com/emu-sh/.oh-my-comma) from modifying `/system` were pruned without explanation
